### ISSUE TITLE
Not working in SignedAPk in Android

### ISSUE THREAD

**Issue by AamilSilawat26** [flutter-clutter/flutter-simple-edge-detection#15]
Not working in **signedAPK** in Android, It is detecting edge very well in release mode, But when I convert release APK to sign APK it is not working and event not detecting edges.

**Comment by flutter-clutter** [comment]
Hey there. Can you extract any error so that we can investigate the issue?

**Comment by AamilSilawat26** [comment]
Hello, I am not getting any kind of error but edge are not detecting in Android SIgnAPK, I have already spend two days for this but still not able to find what's issue is.

**Comment by flutter-clutter** [comment]
That's strange. And the output of the build command does also not give you any clue?
I don't see why there should be any difference between those two.
Are you completely sure that the image you are testing with is supposed to be recognized by the algorithm?

**Comment by AamilSilawat26** [comment]
Yes I have testing this from last 3 days, There is no error performing just edge not detecting in Sign APK you can test it by creating sign APK in Android

**Comment by AamilSilawat26** [comment]
Hello Marc, Any update on above issue ?
Did you checking on this ?

**Comment by flutter-clutter** [comment]
@AamilSilawat26 
Sorry for the late response.
I just checked it. Used this way of creating a signed APK: https://stackoverflow.com/questions/55536637/how-to-build-signed-apk-from-android-studio-for-flutter and copied the APK file on my phone where I installed and executed it. No problem, edges are being detected.

It seems that your issue is not particularly related to this edge detection package.

**Comment by flutter-clutter** [comment]
@AamilSilawat26 
Hey there! Has there been an update? Are you able to build the signed APK now? I hope so!

**Comment by AamilSilawat26** [comment]
Unfortunately not, I am not able to detect edges in Sign APK, as you suggest link and I have tried it, But Still not able to detect edges.

**Comment by cobus-meyer** [comment]
Hi @AamilSilawat26

Were you able to solve this issue? We seem to be stuck with a similar problem.

**Comment by AamilSilawat26** [comment]
> Hi @AamilSilawat26
> 
> Were you able to solve this issue? We seem to be stuck with a similar problem.

Nope Problem is still alive....

**Comment by cobus-meyer** [comment]
@AamilSilawat26 we solved our prolem by using `cv::Scalar` instead of `cv::InputArray` data types.

**Comment by AamilSilawat26** [comment]
> @AamilSilawat26 we solved our prolem by using `cv::Scalar` instead of `cv::InputArray` data types.

In which class or folder you made changes ?

Can you please let me know ?

**Comment by cobus-meyer** [comment]
> > @AamilSilawat26 we solved our prolem by using `cv::Scalar` instead of `cv::InputArray` data types.
> 
> 
> 
> In which class or folder you made changes ?
> 
> 
> 
> Can you please let me know ?

The changes were made in our cpp source files, under ios/Classes/filename.cpp

**Comment by AamilSilawat26** [comment]
> > > @AamilSilawat26 we solved our prolem by using `cv::Scalar` instead of `cv::InputArray` data types.
> > 
> > 
> > In which class or folder you made changes ?
> > Can you please let me know ?
> 
> The changes were made in our cpp source files, under ios/Classes/filename.cpp

I have not found any **cv::InputArray** in any cpp Files !!! 

Can you please give me the exact name of file or which line you have made changes. It would be really appreciate.
